src/work-allocation-2/containers/task-assignment/task-assignment-container.component.ts

```diff
@@ -98,19 +102,28 @@ export class TaskAssignmentContainerComponent implements OnInit, OnDestroy {
   }
 
   public initForm(): void {
-    this.formGroup = new FormGroup({
-      caseWorkerName: new FormControl()
+    this.formGroup = this.fb.group({
+      caseWorkerName: ['', Validators.required]
```

Comment:
`caseworkerName`? There's a fair bit of inconsistency around how `caseworker` appears within the app but it is definitively _one_ word, not two.